python/python/res/job_queue/ext_job.py

```diff
@@ -166,6 +166,10 @@ def arg_types(self):
                 result.append(int)
             elif t == ContentTypeEnum.CONFIG_STRING:
                 result.append(str)
+            elif t == ContentTypeEnum.CONFIG_RUNTIME_FILE:
+                result.append(str)
```

Comment:
To match with #193 this property must now return the `ContentTypeEnum.xxx` types - maybe a new method must be added?